modules/enabling-administrator-interface-access.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
+[id="enabling-administrator-interface-access_{context}"]
+= Enabling administrator interface access
+
+You can change the envoy bootstrap configuration to enable access to the administrator interface.
+
+[IMPORTANT]
+====
+This procedure assumes sufficient knowledge of Knative, as changing envoy bootstrap configuration might result in Knative failure. Red Hat does not support custom configurations that are not tested or shipped with the product.
+====
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+.Procedure
+
+* To enable administrator interface access, locate this configuration in your bootstrapping configuration map:
```

Comment:
```suggestion
. To enable administrator interface access, locate this configuration in your bootstrapping configuration map:
```

Reply:
Since you have a second step below when you say "Apply the configuration in two places", I think this should be a 1. step instead of an unordered item.